---
title: Ein Problem
date: 2021-04-29 15:54:00
resolved: true
resolvedWhen: 2021-04-29 16:10:00
# Possible severity levels: down, disrupted, notice
severity: down
affected:
  - API
  - VPN
section: issue
---

*Was ist passiert*

Um 15:54 haben wir einen Ausfall von API und VPN festgestellt. Nach kurzer Zeit stand die Lösung fest: Bei Reinigungsarbeiten hatte ein Mitarbeiter einen Netzwerkstecker versehentlich gezogen und nicht wieder eingesteckt. Der Fehler war schnell behoben.

---

*Gelöst* – Alle Dienste laufen wieder
*Update* – Wir arbeiten am Problem {{< track "2021-04-29 16:00:00" >}}
